### PR TITLE
Grudgingly disable scalafmt on compile

### PR DESCRIPTION
**Pull request by rossabaker**
This makes me sad, but it's taking my laptop 25 seconds to figure out there's nothing new to compile, on every compile.

I welcome better suggestions, because this is going to trip up new contributors constantly.

### CODE REVIEW

**Comment by aeons** [comment]
Maybe add an sbt task that formats everything and runs tests? Like something to run before `git commit`.

File: project/Http4sPlugin.scala [hunk 1]
```diff
@@ -47,7 +47,16 @@ object Http4sPlugin extends AutoPlugin {
...
+    scalafmt in Test := {
+      (scalafmt in Compile).value
+      (scalafmt in Test).value
+      ()
+    },
+    test in (Test, scalafmt) := {
+      (test in (Compile, scalafmt)).value
+      (test in (Test, scalafmt)).value
+      ()
+    }
```
Comment (rossabaker) [inline project/Http4sPlugin.scala]: I'm actually kind of surprised these aren't defaults.  If we're going to start making people run scalafmt manually, it's nice if we can give them just one command.

**Comment by rossabaker** [comment]
@aeons We already have `validate`, which tests the formatting.  That's the thing people should run before a PR now, but it doesn't format.  Maybe it should, or maybe we could give it a more active name?  (`unsafeValidate` :trollface:)

**Comment by rossabaker** [comment]
Alright.  Let's roll with this, and we can always recalibrate if this proves no good.

**Pull request merged**